site/compiler/people.test.js

```diff
@@ -34,10 +34,10 @@ describe('site/compiler', () => {
         badger: { [a.slug]: a },
       }, createStateNavigator());
 
-      expect(routes.length).to.equal(14);
-      expect(routes[10].filePath).to.equal('about-us/people/index.html');
-      expect(routes[11].filePath).to.equal('about-us/people/category/engineering/index.html');
-      expect(routes[12].filePath).to.equal('about-us/people/category/leadership/index.html');
+      expect(routes.length).to.equal(15);
```

Comment:
-_-

These tests are a pain, but due to the way the router generates pages, there's not much we can do to fix this